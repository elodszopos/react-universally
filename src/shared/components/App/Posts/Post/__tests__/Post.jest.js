import React from 'react';
import { shallow } from 'enzyme';
import { Post } from '../index';

describe('<Post />', () => {
  test('renders', () => {
    const wrapper = shallow(<Post post={{ title: 'postTitle', body: 'postBody' }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
