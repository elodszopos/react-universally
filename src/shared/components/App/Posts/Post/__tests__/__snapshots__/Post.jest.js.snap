exports[`<Post /> renders 1`] = `
<div>
  <HelmetWrapper
    title="Posts - postTitle" />
  <h1>
    postTitle
  </h1>
  <div>
    postBody
  </div>
  <div>
    Foo
  </div>
</div>
`;
